source/frontend/src/components/common/mapFSM/MapStateMachineContext.tsx

```diff
@@ -372,7 +384,12 @@ export const MapStateMachineProvider: React.FC<React.PropsWithChildren<unknown>>
     <MapStateMachineContext.Provider
       value={{
         mapSideBarViewState: state.context.mapSideBarState,
-        isShowingSearchBar: !state.context.mapSideBarState.isOpen && !state.context.isFiltering,
+        isShowingSearchBar:
+          !state.context.mapSideBarState.isOpen &&
```

Comment:
We want to hide the search bar if either the side bar or advanced filter is displayed, or if there is a non-default advanced filter.